app/hooks/search.tsx

```diff
@@ -97,7 +96,9 @@ export const baselineSearch = async (
 const normalize = (question: string) =>
   question
     .toLowerCase()
+    .replace(/\n/g, ' ')
```

Comment:
This could cause problems with things like:
```
{
   title: "this is the end",
   alternatePhrasings: "time to say goodbye",
}
```
when searching for "end time"

Reply:
ah, wouldn't that example be a *good* search result tho?

I know what you mean, but if alternate phrasings should be formulated as questions, accidental match seems unlikely to me TBH - and if it happens, it might not even be undesired in most cases 🤔 

also, more motivation to release production-quality API of semantic search 😅

if we wanted to make production quality keyword search, we should also prioritize match in the title above match in alternate phrasings, and a thousand other manually crafted tweaks that I don't see as the highest priority thing to spend time on for a "temporary" feature (that already lives longer than I expected TBH)

Reply:
that said, feel free to add commits to this PR that you will be be more happy with 😉

Reply:
nah, as long as you're aware of it :D